Library/ViewModels/CommentCellViewModel.swift

```diff
@@ -76,8 +76,12 @@ public final class CommentCellViewModel:
 
     self.flagButtonIsHidden = self.commentAndProject.signal
       .map { _ in
-        AppEnvironment.current.optimizelyClient?
-          .isFeatureEnabled(featureKey: OptimizelyFeature.Key.commentFlaggingDisabled.rawValue) ?? false
+        guard let isFeatureEnabled = AppEnvironment.current.optimizelyClient?
```

Comment:
I think having these split out into functions makes it easier on the eye:

```swift
self.flagButtonIsHidden = self.commentAndProject.signal
  .ignoreValues()
  .map(commentFlaggingEnabled)
  .map(isFalse) // If the feature is enabled, the flag button should be visible.
```

```swift
private func commentFlaggingEnabled() -> Bool {
  AppEnvironment.current.optimizelyClient?
    .isFeatureEnabled(featureKey: OptimizelyFeature.Key.commentFlaggingEnabled.rawValue) ?? true
}
```

Reply:
Just a minor thing, should the nil-coelescing operator be default `false`, because that means the feature is enabled?

Reply:
Updated this although the nil-coalescing is going to default to false because `true` means the feature is enabled now.